src/containers/FileActionMenu.jsx

```diff
@@ -0,0 +1,73 @@
+import styles from '../styles/actionmenu'
+
+import React from 'react'
+import { connect } from 'react-redux'
+import classNames from 'classnames'
+import { translate } from '../lib/I18n'
+import { Item } from 'react-bosonic/lib/Menu'
+import withGestures from 'react-bosonic/lib/withGestures'
+
+import { splitFilename, getClassFromMime } from '../components/File'
+import { getActionableFile } from '../reducers'
+import { downloadFile, hideFileActionMenu } from '../actions'
+
+const Menu = ({ t, file, onDownload }) => {
+  const { filename, extension } = splitFilename(file.name)
+  return (
+    <div className={styles['fil-actionmenu']}>
```

Comment:
I did this because `actionmenu` is one element, I tend to use carets to separate elements, aka: `<app>-<element>-<sub-element>`. But maybe I'm too influenced by BEM here...